deep_quoridor/src/train_sb3.py

```diff
@@ -213,15 +227,15 @@ def eval_action_mask(env_fn, num_games=100, render_mode=None, **env_kwargs):
 if __name__ == "__main__":
     env_fn = quoridor_env
 
-    env_kwargs = {"board_size": 3, "max_walls": 0}
+    env_kwargs = {"board_size": 5, "max_walls": 3}
 
     # Evaluation/training hyperparameter notes:
     # 10k steps: Winrate:  0.76, loss order of 1e-03
     # 20k steps: Winrate:  0.86, loss order of 1e-04
     # 40k steps: Winrate:  0.86, loss order of 7e-06
 
     # Train a model against itself (takes ~20 seconds on a laptop CPU)
-    train_action_mask(env_fn, steps=20_480, seed=0, **env_kwargs)
+    train_action_mask(env_fn, steps=10_480, seed=0, **env_kwargs)
```

Comment:
just curious, any reason for using 10_480 vs a round number like 10_000?

Reply:
no, just copied it from the example. I suppose it's becasue it runs in batches of 2048, so it's 10 batches? And then I wanted to run faster to do tests and I made it "kind of 10k".